### PR TITLE
Remove job image_get

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

File: graph/service.go [hunk 1]
```diff
@@ -74,20 +74,26 @@ func (s *TagStore) CmdSet(job *engine.Job) error {
 	return nil
 }
 
-// CmdGet returns information about an image.
+type ImageDetail struct {
```
Comment (runcom) [inline graph/service.go]: only not sure about this..

Comment (LK4D4) [inline graph/service.go]: I think it is just crap, use `s.ImageLookup` instead of `GetImageDetail` everywhere.

Comment (runcom) [inline graph/service.go]: :+1:

**Comment by LK4D4** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Pull request merged**